_includes/head.html

```diff
@@ -8,7 +8,7 @@
   <link rel="stylesheet" href="{{ site.baseurl }}/css/cayman.css">
   <link rel="stylesheet" href="{{ site.baseurl }}/css/custom1.css">
   <meta name="twitter:card" content="summary_large_image" />
-  <meta property="og:url" content="{{ page.url }}" /
+  <meta property="og:url" content="{{ page.url }}" />
```

Comment:
Unrelated observation: It's good practice to include the website's name in all pages' `<title>`, so while the home page might read `<title>software-architetur.tv</title>`, other pages might use [something like](https://css-tricks.com/whats-the-best-way-to-handle-page-titles/) `<title>{{ page.title }} | software-architetur.tv</title>`.

Reply:
Yes, I originally thought "SoftwareArchitektur" would be nicer but changed my mind. So now it is inconsistent. 😬

Reply:
I added "Software Architektur im Stream" to all titles, see b2509f9c3575c2c64375606a4de4426beaa1e813 . Do you think that's a good idea?

Reply:
Yeah, that looks pretty good to me - makes sense on the home page as well. 👍